intro-presentation/README.md

```diff
@@ -0,0 +1,19 @@
+# DevStaff intro presentation
+The `slides-devstaff-intro.html` file contains the intro presentation of the DevStaff's meetup in an open format. We use [reveal.js](https://github.com/hakimel/reveal.js) as the engine behind the presentation.
+
+The presentation originally set up using the free plan of [slides.com](https://slides.com/).
```

Comment:
minor correction: The presentation was originally set up

Reply:
Fixed! Thanks!